deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,572 @@
+              "affinity": {
+                "nodeAffinity": {
+                  "requiredDuringSchedulingIgnoredDuringExecution": {
+                    "nodeSelectorTerms": [
+                      {
+                        "matchExpressions": [
+                          {
+                            "key": "kubernetes.io/arch",
+                            "operator": "In",
+                            "values": [
+                              "amd64"
+                            ]
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                }
+              }
+            },
+            "sidecars": [
+              {
+                "name": "csi-node-driver-registrar",
+                "repository": "quay.io/k8scsi/csi-node-driver-registrar",
+                "tag": "v1.2.0",
+                "imagePullPolicy": "IfNotPresent"
+              },
+              {
+                "name": "csi-provisioner",
+                "repository": "quay.io/k8scsi/csi-provisioner",
+                "tag": "v1.3.0",
+                "imagePullPolicy": "IfNotPresent"
+              },
+              {
+                "name": "csi-attacher",
+                "repository": "quay.io/k8scsi/csi-attacher",
+                "tag": "v1.2.1",
+                "imagePullPolicy": "IfNotPresent"
+              },
+              {
+                "name": "livenessprobe",
+                "repository": "quay.io/k8scsi/livenessprobe",
+                "tag": "v1.1.0",
+                "imagePullPolicy": "IfNotPresent"
+              }
+            ]
+          }
+        }
+      ]
+spec:
+  displayName: "Operator for IBM block storage CSI driver"
+  description: |
+    **IBM block storage CSI driver** is a Container Storage Interface (CSI) Driver for IBM block storage systems which enables container orchestrators to manage the life cycle of persistent storage.
+
+    This is the official operator to deploy and manage IBM block storage CSI driver.
+
+    Supported container platforms:
+      - OpenShift v4.2
+      - OpenShift v4.3
+      - Kubernetes v1.14
+      - Kubernetes v1.15
+      - Kubernetes v1.16
+
+    Supported IBM storage systems:
+      - IBM FlashSystem 9100
+      - IBM Spectrum Virtualize
+      - IBM Storwize
+      - IBM FlashSystem A9000/R
+      - IBM DS8880
+      - IBM DS8900
+
+    Supported operating systems:
+      - RHEL 7.x (x86 architecture)
+      - RHCOS
+
+    Full documentation can be found on the [IBM knowledge center](https://www.ibm.com/support/knowledgecenter/SSRQ8T).
+
+
+    ## Prerequisites
+
+    ### Preparing worker nodes
+    Perform these steps for each worker node in OpenShift:
+
+    #### 1. Install Linux packages to ensure Fibre Channel and iSCSI connectivity.
+    Skip this step if the packages are already installed.
+
+    RHEL 7.x:
+    ```bash
+    $ yum -y install iscsi-initiator-utils   # only if iSCSI connectivity is required
```

Comment:
Why is the `o` in `only` in lower case? Where is this copied from? the source might be outdated

Reply:
Is it a draft, the readme will be updated later for sure. You can review it if you want.

Reply:
Could you write somewhere that this is only a draft/TODO: overwrite with final README version?

Reply:
I think it is unnecessary, it will be reviewed by Rivka and the team before every release.

Reply:
I don't think Rivka reviews this file, she updates the knowledge center.
And most of the team don't know this file, and it is not covered in our automation.
If it needs to be reviewed before every release, then we need to know where is this file mentioned (ticket/checklist).
If we leave it like this, it will not look like a draft, it will look as finished.
Why do we need an unreviewed draft (that looks finished) in the develop branch anyway?

Reply:
We need a ticket for README + CSV yaml review (separate from CSI-1205)

Reply:
> I don't think Rivka reviews this file, she updates the knowledge center.
> And most of the team don't know this file, and it is not covered in our automation.
> If it needs to be reviewed before every release, then we need to know where is this file mentioned (ticket/checklist).
> If we leave it like this, it will not look like a draft, it will look as finished.
> Why do we need an unreviewed draft (that looks finished) in the develop branch anyway?

I'm not kidding if I say that Rivka has reviewed this file. You can ask her, she will tell you.
Maybe Shay forgot to tell you, when me or Shay want to update something in UT, we will update both files (the readme and here). We are thinking to reduce the duplicated work, but no good idea yet.
I will create ticket to update it, but it is too early to do it now, because things are changing.

Reply:
I see that she has reviewed it [in the past](https://github.com/IBM/ibm-block-csi-operator/pull/25/files/4f8e22049190d669448b898d20542cbf3dab32fa#r342440541), I meant that she is not on this review, so once we have a ticket number here, we can resolve this.

Reply:
created CSI-1261